### PR TITLE
PPM Closeout: Entitlement value

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Show service member their entitlement or a warning message if they are missing a weight ticket

## Reviewer Notes
- temporarily removed the delete buttons on the review page because we do not have that functionality yet

## Setup
`make client_run`
`make server_run`

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/166191650) for this change

## Screenshots
image
image

### CODE REVIEW

File: src/scenes/Moves/Ppm/PaymentReview/ExpenseTicketListItem.jsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { string } from 'prop-types';
-import deleteButtonImg from 'shared/images/delete-doc-button.png';
+// import deleteButtonImg from 'shared/images/delete-doc-button.png';
```
Comment (ralren) [inline src/scenes/Moves/Ppm/PaymentReview/ExpenseTicketListItem.jsx]: Let's delete these commented out lines here and below.

File: src/scenes/Moves/Ppm/PaymentReview/WeightTicketListItem.jsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import faExclamationCircle from '@fortawesome/fontawesome-free-solid/faExclamati
 import carImg from 'shared/images/car_mobile.png';
 import boxTruckImg from 'shared/images/box_truck_mobile.png';
 import carTrailerImg from 'shared/images/car-trailer_mobile.png';
-import deleteButtonImg from 'shared/images/delete-doc-button.png';
+// import deleteButtonImg from 'shared/images/delete-doc-button.png';
```
Comment (ralren) [inline src/scenes/Moves/Ppm/PaymentReview/WeightTicketListItem.jsx]: I believe this also needs to be deleted along with the line below.

File: src/scenes/Moves/Ppm/utility.js [hunk 3]
```diff
@@ -21,3 +22,20 @@ export const formatExpenseDocs = expenseDocs => {
     paymentMethod: expense.payment_method,
   }));
 };
+
+export const calcNetWeight = documents => {
+  console.log(
```
Comment (ralren) [inline src/scenes/Moves/Ppm/utility.js]: Do we still want to console log this?

**Comment by LeDeep** [comment]
Other than the comments from Ren, I think it looks pretty good.

File: src/scenes/Moves/Ppm/PaymentReview/ExpenseTicketListItem.jsx [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
-import deleteButtonImg from 'shared/images/delete-doc-button.png';
 
 const ExpenseTicketListItem = ({ amount, type, paymentMethod, showDelete }) => (
   <div className="ticket-item">
     <div className="expense-li-item-container">
       <h4>
         {type} - ${amount}
       </h4>
-      {showDelete && <img alt="delete document button" onClick={() => console.log('lol')} src={deleteButtonImg} />}
+      {/* {showDelete && <img alt="delete document button" onClick={() => console.log('lol')} src={deleteButtonImg} />} */}
```
Comment (ralren) [inline src/scenes/Moves/Ppm/PaymentReview/ExpenseTicketListItem.jsx]: Please delete.

File: src/scenes/Moves/Ppm/PaymentReview/WeightTicketListItem.jsx [hunk 5]
```diff
@@ -42,7 +41,7 @@ const WeightTicketListItem = ({
         <h4>
           {vehicle_nickname} ({formatToOrdinal(num + 1)} set)
         </h4>
-        {showDelete && <img alt="delete document button" onClick={() => console.log('lol')} src={deleteButtonImg} />}
+        {/* {showDelete && <img alt="delete document button" onClick={() => console.log('lol')} src={deleteButtonImg} />} */}
```
Comment (ralren) [inline src/scenes/Moves/Ppm/PaymentReview/WeightTicketListItem.jsx]: Delete this too.

**Pull request merged**